gordo_components/dataset/datasets.py

```diff
@@ -98,13 +99,18 @@ def get_data(self) -> Tuple[pd.DataFrame, Optional[pd.DataFrame]]:
             to_ts=self.to_ts,
             tag_list=list(set(self.tag_list + self.target_tag_list)),
         )
-        data: pd.DataFrame = self.join_timeseries(
-            series_iter,
-            self.from_ts,
-            self.to_ts,
-            self.resolution,
-            aggregation_methods=self.aggregation_methods,
-        )
+
+        # Resample if we have a resolution set, otherwise simply join the series.
+        if self.resolution is None:
```

Comment:
Is it not more pythonic to do `if self.resolution`...`else`? Then empty string is also regarded as Falsy, just as `False` and `None`